docs/user/mobile-applications.rst

```diff
@@ -195,6 +195,44 @@ In this screen, users can choose which notification types they want to receive.
   .. image:: native-applications/iOS/settings_push_detail.png
             :width: 300
 
+
+Modeling Exercises
+^^^^^^^^^^^^^^^^^^
+Users can participate in modeling exercises, view their submissions and also view the feedback they received in the Artemis iOS app.
+
+- Participate in modeling exercise:
```

Comment:
```suggestion
Participate in modeling exercise
""""""""""""""""""""""""""""""""
```
Possible alternative: The items of the list describe separate features and could be grouped into paragraphs instead.